src/work-allocation-2/components/priority-field/priority-field.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+
+import { TaskPriority } from '../../enums';
+
+@Component({
+  selector: 'exui-priority-field',
+  templateUrl: './priority-field.component.html'
+})
+export class PriorityFieldComponent {
+
+  /**
+   * The current date being examined
+   */
+  @Input() public dueDate: Date;
+
+  public get priority(): TaskPriority {
+    const d = new Date();
```

Comment:
could this d be more descriptive? like 'today'?

Reply:
Done